database/seeders/ApiEndpointsSeeder.php

```diff
@@ -5130,6 +5130,34 @@ private function seedSummitEndpoints()
                     sprintf(SummitScopes::WritePresentationSlidesData, $current_realm)
                 ],
             ],
+            // presentation speakers
+            [
+                'name' => 'add-presentation-speaker',
+                'route' => '/api/v1/summits/{id}/presentations/{presentation_id}/speakers/{speaker_id}',
+                'http_method' => 'POST',
```

Comment:
this need the authz groups
admin, super admin etc

Reply:
done